walkthroughs/2-fuse-aggregator-and-api-management/walkthrough.adoc

```diff
@@ -257,41 +257,41 @@ Verify that you followed each step in the procedure above.  If you are still hav
 === Setting Fuse Aggregation App Endpoint Limits
 
 . Create a new *Application Plan*:
-.. Select *Applications > Application Plans* from the side navigation.
-.. Select *Create Application Plan*.
+.. Click *Applications > Application Plans* from the side navigation.
+.. Click *Create Application Plan*.
 .. Enter the following for *Name* and *System name*:
 +
 [subs="attributes+"]
 ----
 {fuse-flights-aggregator-app-name}
 ----
 .. Leave the other fields with their default values.
-.. Select *Create Application Plan*. You will be redirected to the *Application Plans* screen.
-.. Select the *Publish* button, beside your plan list item, to publish the Plan.
+.. Click *Create Application Plan*. You are redirected to the *Application Plans* screen.
+.. Click the *Publish* button.  Beside your plan list item, click the Publish button to publish the Plan.
```

Comment:
this seems awkward

Reply:
Covered in https://issues.jboss.org/browse/INTLY-2440